styleguide/derek/solutions/solutions-usage.ejs

```diff
@@ -199,12 +199,12 @@
     Headline - 30. <br/>
     Description - 200.<br/>
     List Limit: 12 list items</h6>
-  <p>The Features pattern should be used to showcase important or key features of the product or service. This pattern should be placed on a background color and the feature copy should be appraochable and not overly technical. There is also the list pattern which can be used to add additional features so that the features section doesn't get overly heavy.</p>
+  <p>The Features pattern should be used to showcase important or key features of the product or service. This pattern should be placed on a background color ,of either white or light gray, and the feature copy should be appraochable and not overly technical. There is also the list pattern which can be used to add additional features so that the features section doesn't get overly heavy.</p>
   <div class="row">
     <div class="col-md-6">
       <h4 class="red">DO</h4>
       <ul class="checks">
-        <li>Place on background color</li>
+        <li>Place on background color (only either white or light gray)</li>
```

Comment:
Can we change this to remove "place on a background color" since white isnt a BG color. and then add into the DONT side "dont place on a BG color darker than light-gray"